lms/djangoapps/certificates/tests/test_api.py

```diff
@@ -104,6 +110,7 @@ def setUp(self):
         )
 
         self.request_factory = RequestFactory()
+        self.global_staff = GlobalStaffFactory()
```

Comment:
You are using it in only one method move this to method instead of creating in staff.